src/acl.js

```diff
@@ -0,0 +1,53 @@
+class ACL {
+  constructor (keystore, key, keys = []) {
+    this._keystore = keystore
+    this._key = key
+    this._keys = keys && Array.isArray(keys) ? keys : keys ? [keys] : []
+  }
+
+  getSigningKey () {
+    return this._key
+  }
+
+  getPublicSigningKey (format = 'hex') {
+    return this._key.getPublic(format)
+  }
+
+  canAppend (signingKey) {
```

Comment:
If we pass functions from orbit-db access controller to it, we wouldn't need this anymore